backend/bot/views.py

```diff
@@ -31,50 +32,66 @@ def post(self, request):
                 {"success": False, "message": "please enter a valid openAi key"},
                 status=status.HTTP_400_BAD_REQUEST,
             )
-        bot_instance = serializer.create() 
-        message, response = add_goals(serializer.validated_data, bot_instance)
-        if not response:
-            return Response(
-                {"success": False, "message": message},
-                status=status.HTTP_400_BAD_REQUEST,
-            )
+        bot_instance = serializer.create()
+        validate_data = serializer.validated_data
+        if "tag_type" in validate_data:
+            tag_type_objects = [TagType(**{**tag_goal, "bot": bot_instance}) for tag_goal in validate_data['tag_type']]
```

Comment:
https://stackoverflow.com/questions/68099778/bulk-create-using-listcreateapiview-and-listserializer

Try this for performing these operations